operator/cmd/root.go

```diff
@@ -116,9 +116,12 @@ var operatorCell = hive.NewCell(
 )
 
 func registerOperatorHooks(lc fx.Lifecycle, clientset k8sClient.Clientset, shutdowner fx.Shutdowner) {
-	k8s.SetClients(clientset, clientset.Slim(), clientset, clientset)
 	initEnv()
 
+	if !operatorOption.Config.DisableK8sServices {
```

Comment:
I'm curious if you have any thoughts on the rationale between just using `k8s.IsEnabled()` here vs. the new option. Is there a meaningful difference?